### DISCUSSION TITLE
Meteor-M2 3 not working!

### DISCUSSION THREAD

**Discussion by Itsisme** [jekhokie/raspberry-noaa-v2#918] [General]
Hello,

I have tried a few things but never have I seen any meteor-m2 3 decoding yet.
Is there anyone who has got this working and can publish the correct settings (and/or code changes)??

The github is only reflecting the meteor-m2 2 ( ancient one :) )
That in the past when the sat was still operating well worked.

Would be nice to get the new meteor-m2 3 working.

![noaav2m23](https://github.com/jekhokie/raspberry-noaa-v2/assets/126970761/5c59a792-486f-483f-98d5-236653a2aa58)

As you can see NOAA goes well.. Meteor fails.. 

![noaav2noa](https://github.com/jekhokie/raspberry-noaa-v2/assets/126970761/3b14fb29-f911-475d-9007-1902534e90d3)

Chrs..

**Comment by nxugget** [comment]
Hi, I think I know why. I've created a pull request because the frequency of **meteor m2-3** is set to **137.9000** but has changed in 2023 to **137.1000**. You can change it by yourself by editing the **common.sh** file et **line 54**.

**Reply by Itsisme** [reply]
Super I changed it lets see what is does now.
I noticed meteor m2-4 also being on 137.100 (and also not decoding yet)

![noaav2-index](https://github.com/jekhokie/raspberry-noaa-v2/assets/126970761/94efbcde-52bb-4eb7-8d60-ad1df4207313)

What about this screen any fix for that?

Cheers
